### PR TITLE
fix: make cardboard button psudeo fullscreen on iOS

### PR DESCRIPTION
**Pull request by BrandonOCasey**
fix: hide cardboard button when fullscreen on Android
fix: typo that caused an error in the console

### CODE REVIEW

File: src/cardboard-button.js [hunk 1]
```diff
@@ -4,11 +4,79 @@ import videojs from 'video.js';
...
+    }
+  }
+
   buildCSSClass() {
     return `vjs-button-vr ${super.buildCSSClass()}`;
   }
-  handleClick() {
-    window.dispatchEvent(new window.Event('vrdisplayactivate'));
+
+  handleActivate_() {
```
Comment (gkatsev) [inline src/cardboard-button.js]: One thing to look into is whether `player.enterFullWindow` and `player.exitFullWindow` work for this usecase.

Comment (BrandonOCasey) [inline src/cardboard-button.js]: I tried and it actually exists cardboard mode right away

Comment (BrandonOCasey) [inline src/cardboard-button.js]: testing

Comment (BrandonOCasey) [inline src/cardboard-button.js]: does not appear to work

File: src/cardboard-button.js [hunk 2]
```diff
@@ -4,11 +4,79 @@ import videojs from 'video.js';
 const Button = videojs.getComponent('Button');
 
 class CardboardButton extends Button {
+  constructor(player, options) {
+    super(player, options);
+
+    window.addEventListener('vrdisplayactivate', () => this.handleActivate_());
+    window.addEventListener('vrdisplaydeactivate', () => this.handleDeactivate_());
```
Comment (misteroneill) [inline src/cardboard-button.js]: Should probably make sure we clean up after ourselves in a `dispose` listener.

Comment (BrandonOCasey) [inline src/cardboard-button.js]: good call, going to take some time to do that in `player.js` too

Comment (BrandonOCasey) [inline src/cardboard-button.js]: scratch that its going to take a revamp of `player.js` which I would rather do in another PR
